distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,347 @@
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add an association
+|Configurable
+|no
+
+|===
+
+.[[service_bindings]]Services
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Service Name
+|This service name
+|string
+|
+
+|Service Description
+|Short description for this service
+|string
+|
+
+|Service Version
+|This service version
+|string
+|
+
+|Service Type
+|Identifies the type of service this is by a URN.
+|string
+|
+
+4+^|Bindings (Click *Add* to add a service binding)
+
+|Binding Name
+|This binding name
+|String
+|yes
+
+|Binding Description
+|Short description for this binding
+|String
+|
+
+|Binding Version
+|This binding version
+|
+|
+
+|Access URL
+|The url used to access this binding
+|
+|
+
+|Service Binding Type
+|The binding type for the service
+|
+|
+
+|URL Property Key
+|Property that the accessURI value should be put into for source creation
+|
+|
+
+4+|
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add an association
+|Configurable
+|no
+
+|===
+
+.Organizations Tab (click *Add* to add an organization)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Organization Name
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|Primary contact number for this organization
+|
+|no
+
+|Email
+|Primary contact email for this organization
+|
+|no
+
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Contacts (click *Add* button to add contact info)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Contact Title
+|Contact Title
+|String
+|yes
+
+|Contact First Name
+|Contact First Name
+|String
+|yes
+
+|Contact Last Name
+|Contact Last Name
+|String
+|yes
+
+|Address
+|Address for listed contact
+|String
+|minimum one
+
+|Phone number
+|Contact phone number
+|
+|minimum one
+
+|Email
+|Contact email
+|String
+|minimum one
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Collections (Click *Add* to add Content Collection(s))
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Content Name
+|Name for this metadata content
+|string
+|yes
+
+|Content Description
+|Short description for this metadata content
+|string
+|no
+
+|Content Object Type
+|The kind of content object this will be. Default value should be used in most cases.
+|string
+|yes
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add an association
+|Configurable
+|no
+
+|===
+
+===== Adding a Service Binding
```

Comment:
This header works, but it's a little unclear. I think it would be better put as `Adding a Service Binding to a Node`.

Also this step: `+* Enter (at minimum) a *Node Name* under *General Information* tab).` is very confusing. 
The previous steps seem to imply that the node in question is already created (which means this step is not necessary).